subworkflows/local/blobtools.nf

```diff
@@ -0,0 +1,36 @@
+//
+//
+// Generate .yaml config file from meta, then run (1) create, (2) add, and (3) add_summary_to_metadata
+//
+//
+
+nextflow.enable.dsl = 2
+
+include { GENERATE_CONFIG } from '../../modules/local/generate_config'
+
+workflow BLOBTOOLS {
+    take:
+
+    //  Tuple [meta, fasta]:
+    fasta
+
+    main:
+
+    ch_versions = Channel.empty()
+
+    //
+    // Generate config file
+    //
+    GENERATE_CONFIG (
+    fasta.map { it[0] }
```

Comment:
I would recommend pass the whole channel along with the genome file, even though you will not use it.